Cargo.toml

```diff
@@ -8,10 +8,10 @@ members = [
 
 [workspace.package]
 edition = "2021"
-rust-version = "1.76.0"
+rust-version = "1.79.0"
```

Comment:
Why don't we use latest stable version: `1.82.0`?
```
rustup update stable
rustc --version     
rustc 1.82.0 (f6e511eec 2024-10-15)
```

Reply:
rust 1.82 won't work because of https://github.com/stellar/rs-soroban-sdk/pull/1371 . We could do 1.81 but i just decided to go with the minimum version that will probably work instead of the newest possible one as long as we don't need any features from the later version.